index.js

```diff
@@ -2292,6 +2292,24 @@
                R.pipe(R[name], Just, R.filter(R.gte(_, 0))));
   };
 
+  //# append :: a -> Array a -> Array a
```

Comment:
This is still in the List section. This function is Array-specific so belongs in the Array section. Let's do this:

```diff
   //. ### Array

+  //# append :: a -> Array a -> Array a
+  //.
+  //. Takes a value of any type and an array of values of that type, and
+  //. returns the result of appending the value to the array.
+  //.
+  //. ```javascript
+  //. > S.append(3, [1, 2])
+  //. [1, 2, 3]
+  //.
+  //. > S.append([3, 4], [[1], [2]])
+  //. [[1], [2], [3, 4]]
+  //. ```
+  S.append =
+  def('append',
+      {},
+      [a, $.Array(a), $.Array(a)],
+      function(x, xs) { return xs.concat([x]); });
+
   //# find :: (a -> Boolean) -> Array a -> Maybe a
```